backend/src/asiakirja/asiakirjaEmailService.ts

```diff
@@ -19,11 +22,12 @@ export class AsiakirjaEmailService {
     }
 
     assertIsDefined(options.kielitiedot?.ensisijainenKieli);
-    options.kieli = options.kielitiedot?.ensisijainenKieli;
+    assert(kieliIsTranslatable(options.kielitiedot?.ensisijainenKieli));
+    options.kieli = options.kielitiedot?.ensisijainenKieli as KaannettavaKieli;
     const email = new Kutsu21(options).createEmail();
 
-    if (options.kielitiedot?.toissijainenKieli) {
-      options.kieli = options.kielitiedot?.toissijainenKieli;
+    if (options.kielitiedot?.toissijainenKieli && kieliIsTranslatable(options.kielitiedot.toissijainenKieli)) {
```

Comment:
Ehdotus: kieliIsTranslatable voisi olla isKieliTranslatable, jotta on ehkä helpompi lukea. Se voisi hyvöksyä myös null | undefined parametrin, joka palauttaa silloin falsen. Tällöin ei tartte tuota iffin alkuosaa ollenkaan tuossa.